echopype/convert/set_groups_azfp.py

```diff
@@ -167,18 +168,21 @@ def set_platform(self) -> xr.Dataset:
         """Set the Platform group."""
         platform_dict = {"platform_name": "", "platform_type": "", "platform_code_ICES": ""}
         unpacked_data = self.parser_obj.unpacked_data
+        time2 = self.parser_obj.ping_time
+        if np.isnan(unpacked_data["pressure"]).all():
+            depth = [np.nan] * len(time2)
+            water_level = [np.nan]
+        else:
+            depth = -1 * depth_from_pressure(
+                unpacked_data["pressure"], latitude=30, atm_pres_surf=0
+            )
+            water_level = [0.0]
```

Comment:
`water_level` is a scalar variable, so its value should not be a list.
```suggestion
            water_level = 0.0
```

Reply:
Updated!